### PR TITLE
[adaptive_scaffold] Release `adaptive_scaffold` package

### PR DESCRIPTION
**Pull request by gspencergoog**
## Description

This marks the `adaptive_scaffold` package for publishing.

### CODE REVIEW

File: packages/adaptive_scaffold/pubspec.yaml
Comment (stuartmorgan) [inline packages/adaptive_scaffold/pubspec.yaml:6]: Do we actually need the `masonry_grid` dependency here? We try to minimize dependencies on third-party packages in things we publish under the verified publisher. It's not at all clear that the project is actively maintained, which would put us in an unfortunate position if, say, a Flutter update broke it and we couldn't get fixes landed for it.

It seems like there's very little code in the package. Is it really something we can't just implement directly?

Comment (gspencergoog) [reply packages/adaptive_scaffold/pubspec.yaml:6]: In another PR (https://github.com/flutter/packages/pull/2558), I've replaced it with my own implementation of a similar layout.  Once that lands, I'll rebase this on top of HEAD again, and we can publish.

**Review by a-wallen — APPROVED** [review]
LGTM, but we should wait on @stuartmorgan's review.

**Review by stuartmorgan — APPROVED** [review]
LGTM! Thanks for eliminating the extra dependency.

**Comment by gspencergoog** [comment]
Crud.  Someone else beat us to the name, so we'll have to rename this package. https://pub.dev/packages/adaptive_scaffold

**Comment by gspencergoog** [comment]
I guess nobody checked to see if the name was already taken.

**Comment by stuartmorgan** [comment]
Huh, I'm surprised the publish check didn't fail.

**Comment by gspencergoog** [comment]
Yeah, me too.

**Comment by stuartmorgan** [comment]
Oh, there was never a version 0.0.3 of that package. `--dry-run` docs say "Validate but do not publish the package." so since it's just validating the package itself, there's probably no auth check to see if we are valid uploaders.

In other words, there's nothing inherently invalid about uploading this code as `adpative_scaffold` 0.0.3... we just aren't authorized to do it.

**Pull request merged**
